Sources/StreamChat/Database/DTOs/MemberModelDTO.swift

```diff
@@ -198,13 +198,15 @@ extension ChatChannelMember {
             extraData = [:]
         }
 
-        var memberExtraData: [String: RawJSON] = [:]
-        if let dtoMemberExtraData = dto.extraData {
-            do {
-                memberExtraData = try JSONDecoder.default.decode([String: RawJSON].self, from: dtoMemberExtraData)
-            } catch {
-                memberExtraData = [:]
-            }
+        let memberExtraData: [String: RawJSON]
```

Comment:
Another interesting thing that we could try to improve is that `ChatMember` and `ChatUser` are both classes. And I think we should be able to convert them to structs without API changes. Could be interesting to see if there are any performance improvements by doing so 🤔

Reply:
why do you think structs would improve performance? Probably only the SDK size would increase.

Reply:
ChatUser.init itself is visible in the time profiler, but it is like 1 ms which is nothing compared to other things happening. At some point it would be nice to only use structs for models (consistent).

Reply:
Usually  structs a lightweight to create and destroy, so it should improve things, but maybe not significantly. @laevandus We not only need to check the init, but also how much time is spend destroying existing objects 🤔

Reply:
Based on time profiler I don't see this popping up really, a couple of ms for `ChatUser.__deallocating_deinit` and `ChatChannelMember.__deallocating_deinit`. We should do that for consistency, but in a separate ticket. No real performance improvement here.

Reply:
Okay nice 👍